test/extensions/transport_sockets/alts/tsi_handshaker_test.cc

```diff
@@ -86,15 +86,15 @@ TEST_F(TsiHandshakerTest, DoHandshake) {
 
   tsi_peer client_peer;
   EXPECT_EQ(TSI_OK, tsi_handshaker_result_extract_peer(client_result.get(), &client_peer));
-  EXPECT_EQ(1, client_peer.property_count);
+  EXPECT_EQ(2, client_peer.property_count);
   EXPECT_STREQ("certificate_type", client_peer.properties[0].name);
   absl::string_view client_certificate_type{client_peer.properties[0].value.data,
                                             client_peer.properties[0].value.length};
   EXPECT_EQ("FAKE", client_certificate_type);
 
   tsi_peer server_peer;
   EXPECT_EQ(TSI_OK, tsi_handshaker_result_extract_peer(server_result.get(), &server_peer));
-  EXPECT_EQ(1, server_peer.property_count);
+  EXPECT_EQ(2, server_peer.property_count);
```

Comment:
I think this change is right, but double checking with @yihuazhang

Reply:
Yes, it is correct.  Since the test used `fake_transport_security` implementation, based on https://github.com/grpc/grpc/blob/master/src/core/tsi/fake_transport_security.cc#L495, it creates two peer properties: certificate type and security level.

Reply:
Thanks, SGTM.

Reply:
Thanks for validating that test change @htuch and @yihuazhang.

The issue with CI now are two failures. One is a ALTS test timeout and one is a gRPC failure. I suspect that both are related to the same thing and possibly with gRPC.

The ALTS test timeout occurs at https://github.com/envoyproxy/envoy/blob/master/test/extensions/transport_sockets/alts/alts_integration_test.cc#L264-L270

```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //test/extensions/transport_sockets/alts:alts_integration_test
-----------------------------------------------------------------------------
WARNING: Perftools heap leak checker is active -- Performance may suffer
[==========] Running 12 tests from 6 test suites.
[----------] Global test environment set-up.
[----------] 2 tests from IpVersions/AltsIntegrationTestValidPeer
[ RUN      ] IpVersions/AltsIntegrationTestValidPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv4
[       OK ] IpVersions/AltsIntegrationTestValidPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv4 (667 ms)
[ RUN      ] IpVersions/AltsIntegrationTestValidPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv6
[       OK ] IpVersions/AltsIntegrationTestValidPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv6 (441 ms)
[----------] 2 tests from IpVersions/AltsIntegrationTestValidPeer (1108 ms total)

[----------] 2 tests from IpVersions/AltsIntegrationTestEmptyPeer
[ RUN      ] IpVersions/AltsIntegrationTestEmptyPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv4
[       OK ] IpVersions/AltsIntegrationTestEmptyPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv4 (466 ms)
[ RUN      ] IpVersions/AltsIntegrationTestEmptyPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv6
[       OK ] IpVersions/AltsIntegrationTestEmptyPeer.RouterRequestAndResponseWithBodyNoBuffer/IPv6 (464 ms)
[----------] 2 tests from IpVersions/AltsIntegrationTestEmptyPeer (930 ms total)

[----------] 2 tests from IpVersions/AltsIntegrationTestClientInvalidPeer
[ RUN      ] IpVersions/AltsIntegrationTestClientInvalidPeer.ClientValidationFail/IPv4
-- Test timed out at 2020-10-06 00:11:04 UTC --
```

The gRPC error occurs at https://github.com/envoyproxy/envoy/blob/master/test/common/grpc/grpc_client_integration_test.cc#L74-L91

```
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] IpVersionsClientType/GrpcClientIntegrationTest.HttpNon200Status/IPv4_GoogleGrpc, where GetParam() = (4-byte object <00-00 00-00>, 4-byte object <01-00 00-00>)
[  FAILED  ] IpVersionsClientType/GrpcClientIntegrationTest.HttpNon200Status/IPv6_GoogleGrpc, where GetParam() = (4-byte object <01-00 00-00>, 4-byte object <01-00 00-00>)
```
Are you aware of a change in gRPC (or ALTS) that would unblock these two tests?